openfisca_web_api_preview/app.py

```diff
@@ -128,7 +129,10 @@ def apply_headers(response):
     @app.after_request
     def track_requests(response):
         if tracker:
-            tracker.track(request.url)
+            if 'dnt' in request.headers and request.headers['dnt'] == '1':
```

Comment:
`request.headers` seems to have the same API than a python `dict`.
You can thus just use `if request.headers.get('dnt')`.

If the 'DNT' header is not set, the value will be None, evaluated to false.
If the 'DNT' header is set to 0, the value will be 0, evaluated to false.
if the 'DNT' header is set to 1, true, TRUE, the value will be evaluated to true.